### ISSUE TITLE
TODO List ❗ 

### ISSUE THREAD

**Issue by JordanLeich** [JordanLeich/Ultimate-Calculator#1]
1. Fix y=mx+b function
2. Add any calculations or converters to help complete this project!

**Comment by aryanag7** [comment]
Hey, I'm a beginner in open source and would like to contribute to your project, Can u let me know how should I get started?

**Comment by JordanLeich** [comment]
> Hey, I'm a beginner in open source and would like to contribute to your project, Can u let me know how should I get started?

Sure, first you must fork the project or whichever file you would like to edit and then once you are finished making your changes, make a pull request and I will review the changes you made it and then your changes will be added to the project.

**Comment by BadrBelkadi** [comment]
Hi! I like your project ;) . Have you thought about using an API to update the conversions?

**Comment by JordanLeich** [comment]
> Hi! I like your project ;) . Have you thought about using an API to update the conversions?

Thanks, I appreciate it a lot 😀  
I'm not sure exactly how to use an API in python since I've haven't learned it yet but I'm sure it would be helpful. If you want to could try adding an API to this project and I'll take a look at it.

**Comment by hanadrizz** [comment]
Seems interesting! I'll contribute, working on the time converter now!

**Comment by JordanLeich** [comment]
> Seems interesting! I'll contribute, working on the time converter now!

I've merged your pull request, looks all good! Thanks again.

**Comment by hanadrizz** [comment]
if theres any other things to add, I'll see if I'll add that

**Comment by JordanLeich** [comment]
Hey @BadrBelkadi do you still have the old .ui files for the GUI? If so please add them back into the project as we need them for further editing.

**Comment by Some-Person-That-Codes** [comment]
Can I help by adding factorial and double factorial?

**Comment by JordanLeich** [comment]
@Some-Person-That-Codes yes you can.

**Comment by mordy-python** [comment]
Hey Jordan, I was testing out the GUI and I noticed that the buttons are too cramped and small, and the text isn't rendering on them right, I already have a basic arithmetic calculator, can I swap out the two calculators and if you like it, I can work on the other UI components?

**Comment by JordanLeich** [comment]
> Hey Jordan, I was testing out the GUI and I noticed that the buttons are too cramped and small, and the text isn't rendering on them right, I already have a basic arithmetic calculator, can I swap out the two calculators and if you like it, I can work on the other UI components?

Sure thing, the GUI version of this project definitely needs to be updated since it's missing a lot of what's in the text based version.

**Comment by mordy-python** [comment]
I'll get started!

Mordy Waldner

*Blog: https://tech-snippets.web.app <https://tech-snippets.web.app>*
Github: https://github.com/mordy-python/
Chemyst Library: https://pypi.org/project/chemyst/
Chemyst library Docs. https://mordy-python.github.io/chemyst/

**Comment by daniel-suzuki** [comment]
Hey @JordanLeich, is this to-do list up to date?

**Comment by JordanLeich** [comment]
> Hey @JordanLeich, is this to-do list up to date?

Yup, everything is up to date. Feel free to add anything you would like to the project and I'll review it!
